docs/getting-started/install-scylla/install-on-linux.rst

```diff
@@ -46,7 +46,7 @@ Install ScyllaDB
 
             .. code-block:: console
     
-               sudo gpg --homedir /tmp --no-default-keyring --keyring /etc/apt/keyrings/scylladb.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 491c93b9de7496a7
+               sudo gpg --homedir /tmp --no-default-keyring --keyring /etc/apt/keyrings/scylladb.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 491c93b9de7496a7 a43e06657bac99e3
```

Comment:
It should be replaced not added on top :-)

Reply:
but we are using both the old key and the new one -- the old packages were signed with the old key. i think this document applies to old packages as well, am i right?

Reply:
see also https://github.com/scylladb/scylla-pkg/wiki/GPG---keys-lifecycle-and-setup

Reply:
i see.

Reply:
> `a43e06657bac99e3` key is for mater/enterprise 2024.2 and 6.2 (i have updated the page)
> 
> btw, i am not sure if this is the place we should change, maybe @annastuchlik or @tzach can confirm ?

good point. but i think @annastuchlik wanted *move* the package install instructions to this place. see 5895e210fdc3d06d25b256d50491bff41f83eb66 . but yeah, we might want to update the https://www.scylladb.com/download/?platform=ubuntu-22.04&version=scylla-6.2#open-source document accordingly.

Reply:
thank you for your review and approval!